### ISSUE TITLE
Appiphony Lookup Close Icon

### ISSUE THREAD

**Issue by kalusartem** [appiphony/appiphony-lightning-js#98]
I am using the Appiphony JS and the problem with it is the close icon not showing for selected item in the lookup fields. Please help.

Missing icon:
![2018-03-03 2 12 50](https://user-images.githubusercontent.com/28960791/36938253-1bec67f4-1eed-11e8-920c-4e4b1a08ecc4.png)

CODE WITH PROBLEM
`<apex:page showHeader="false" standardStylesheets="false" sidebar="false" applyHtmlTag="false" applyBodyTag="false" docType="html-5.0" >

<html xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">

    <head>
        <meta charset="utf-8" />
        <meta http-equiv="x-ua-compatible" content="ie=edge" />
        <title>Salesforce Lightning Design System Trailhead Module</title>
        <meta name="viewport" content="width=device-width, initial-scale=1" />

        
        <apex:slds/>

        
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
        <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>

        
        <script src="{!URLFOR($Resource.aljs,'/jquery.aljs-all-with-moment.min.js')}"></script>

        <script>
            var j$ = jQuery.noConflict();
        </script>
    </head>

    <body>

        
        <div class="slds-scope">
            <div class="slds-form-element slds-lookup" data-select="single" data-scope="single">
                <label class="slds-form-element__label" for="lookup-01">Objects</label>
                <div class="slds-form-element__control slds-input-has-icon slds-input-has-icon--right">
                    <svg aria-hidden="true" class="slds-input__icon slds-icon-text-default">
                        <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="{!URLFOR($Asset.SLDS, '/assets/icons/utility-sprite/svg/symbols.svg#search')}"></use>
                    </svg>
                    <input id="lookup-01" class="slds-lookup__search-input slds-input" type="text" aria-autocomplete="list" role="combobox" aria-expanded="false" aria-activedescendant="" placeholder="Search Objects"/>
                </div>
            </div>
            <div class="slds-form-element slds-lookup" data-select="single" data-scope="single">
                <label class="slds-form-element__label" for="lookup-02">Things</label>
                <div class="slds-form-element__control slds-input-has-icon slds-input-has-icon--right">
                    <svg aria-hidden="true" class="slds-input__icon slds-icon-text-default">
                        <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="{!URLFOR($Asset.SLDS, '/assets/icons/utility-sprite/svg/symbols.svg#search')}"></use>
                    </svg>
                    <input id="lookup-02" class="slds-lookup__search-input slds-input" type="text" aria-autocomplete="list" role="combobox" aria-expanded="false" aria-activedescendant="" placeholder="Search Things"/>
                </div>
            </div>
        </div>

    </body>

    <script>
    j$(document).ready(function() {
        j$('#lookup-01').lookup({
            items: [
                {
                    id: 'result-1',
                    label: 'Result 1',
                    metaLabel: 'Optional meta description'
                },
                {
                    id: 'result-2',
                    label: 'Result 2',
                    metaLabel: 'Optional meta description'
                },
                {
                    id: 'result-3',
                    label: 'Result 3',
                    metaLabel: 'Optional meta description'
                },
                {
                    id: 'result-4',
                    label: 'Result 4',
                    metaLabel: 'Optional meta description'
                }
            ],
            objectIconUrl: '{!URLFOR($Asset.SLDS, "/assets/icons/standard-sprite/svg/symbols.svg#account")}',
            showSearch: true,
            recentLabel: ''
        });

        var resultsForSecondLookup = [
            {
                id: 'ajax-result-1',
                label: 'AJAX Result 1',
                metaLabel: 'Optional meta description'
            },
            {
                id: 'ajax-result-2',
                label: 'AJAX Result 2',
                metaLabel: 'Optional meta description'
            },
            {
                id: 'ajax-result-3',
                label: 'AJAX Result 3',
                metaLabel: 'Optional meta description'
            },
            {
                id: 'ajax-result-4',
                label: 'AJAX Result 4',
                metaLabel: 'Optional meta description'
            }
        ];

        j$('#lookup-02').lookup({
            items: resultsForSecondLookup,
            objectPluralLabel: 'Things',
            objectLabel: 'Thing',
            useImgTag: false,
            objectIconUrl: '{!URLFOR($Asset.SLDS, "/assets/icons/standard-sprite/svg/symbols.svg#account")}',
            objectIconClass: 'slds-icon-standard-account',
            initialSelection: { id: 'ajax-result-1', label: 'AJAX Result 1' },
            onClickNew: function() {
                console.log('redirect or run something');
            }
        });
    });
    </script>

</html>

</apex:page>`

WORKAROUND:

`<apex:page showHeader="false" standardStylesheets="false" sidebar="false" applyHtmlTag="false" applyBodyTag="false" docType="html-5.0" >
    
    <html xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
        
        
        <head>
            <meta charset="utf-8" />
            <meta http-equiv="x-ua-compatible" content="ie=edge" />
            <title>Salesforce Lightning Design System Trailhead Module</title>
            <meta name="viewport" content="width=device-width, initial-scale=1" />
            
            
            
            <apex:slds />
            
            
            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
            
            
            
                <script src="{!URLFOR($Resource.aljs,'/jquery.aljs-all-with-moment.min.js')}"></script>
            
            <script>
            var j$ = jQuery.noConflict();
            </script>
        </head>
        
        <body>
            
            
            <div class="slds-scope">
                <div class="slds-form-element slds-lookup" data-select="single" data-scope="single">
                    <label class="slds-form-element__label" for="lookup-01">Objects</label>
                    <div class="slds-form-element__control slds-input-has-icon slds-input-has-icon--right">
                        <svg aria-hidden="true" class="slds-input__icon slds-icon-text-default">
                            <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="{!URLFOR($Asset.SLDS, '/assets/icons/utility-sprite/svg/symbols.svg#search')}"></use>
                        </svg>
                        <input id="lookup-01" class="slds-lookup__search-input slds-input" type="text" aria-autocomplete="list" role="combobox" aria-expanded="false" aria-activedescendant="" placeholder="Search Objects"/>
                    </div>
                </div>
                <div class="slds-form-element slds-lookup" data-select="single" data-scope="single">
                    <label class="slds-form-element__label" for="lookup-02">Things</label>
                    <div class="slds-form-element__control slds-input-has-icon slds-input-has-icon--right">
                        <svg aria-hidden="true" class="slds-input__icon slds-icon-text-default">
                            <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="{!URLFOR($Asset.SLDS, '/assets/icons/utility-sprite/svg/symbols.svg#search')}"></use>
                        </svg>
                        <input id="lookup-02" class="slds-lookup__search-input slds-input" type="text" aria-autocomplete="list" role="combobox" aria-expanded="false" aria-activedescendant="" placeholder="Search Things"/>
                    </div>
                </div>
            </div>
            
            <button class="slds-button slds-button_neutral">Test</button>
            
        </body>
        
        <script>
        j$(document).ready(function() {
            j$('#lookup-01').lookup({
                items: [
                    {
                        id: 'result-1',
                        label: 'Result 1',
                        metaLabel: 'Optional meta description'
                    },
                    {
                        id: 'result-2',
                        label: 'Result 2',
                        metaLabel: 'Optional meta description'
                    },
                    {
                        id: 'result-3',
                        label: 'Result 3',
                        metaLabel: 'Optional meta description'
                    },
                    {
                        id: 'result-4',
                        label: 'Result 4',
                        metaLabel: 'Optional meta description'
                    }
                ],
                objectIconUrl: '{!URLFOR($Asset.SLDS, "/assets/icons/standard-sprite/svg/symbols.svg#account")}',
                showSearch: true,
                recentLabel: ''
            });
            
            var resultsForSecondLookup = [
                {
                    id: 'ajax-result-1',
                    label: 'AJAX Result 1',
                    metaLabel: 'Optional meta description'
                },
                {
                    id: 'ajax-result-2',
                    label: 'AJAX Result 2',
                    metaLabel: 'Optional meta description'
                },
                {
                    id: 'ajax-result-3',
                    label: 'AJAX Result 3',
                    metaLabel: 'Optional meta description'
                },
                {
                    id: 'ajax-result-4',
                    label: 'AJAX Result 4',
                    metaLabel: 'Optional meta description'
                }
            ];
            
            j$('#lookup-02').lookup({
                items: resultsForSecondLookup,
                objectPluralLabel: 'Things',
                objectLabel: 'Thing',
                useImgTag: false,
                objectIconUrl: '{!URLFOR($Asset.SLDS, "/assets/icons/standard-sprite/svg/symbols.svg#account")}',
                objectIconClass: 'slds-icon-standard-account',
                initialSelection: { id: 'ajax-result-1', label: 'AJAX Result 1' },
                onClickNew: function() {
                    console.log('redirect or run something');
                },
                onChange: function() {
                    chankeLookup();
                }
            });
            
            //called on lookup change
            function chankeLookup(){
                setCross();
            };
            
            //called from chankeLookup()
            function setCross(){
                j$( "use" ).each(function( index ) {
                    //console.log( index + ": " + j$( this ).text() );
                    console.log( 'Test' );
                    
                    console.log( this.href.baseVal );
                    if(this.href.baseVal.includes('close')){
                        this.href.baseVal = '{!URLFOR($Asset.SLDS, "/assets/icons/utility-sprite/svg/symbols.svg#close")}';
                    }
                });
            }
            
            //button test
            j$("[class*='slds-button']").click(function( index ) {
                
                console.log( 'Test Button' );
                
            });
            
        });
        </script>
        
        
    </html>
</apex:page>`
